pkg/receive/multitsdb.go

```diff
@@ -97,12 +97,18 @@ func NewMultiTSDB(
 type localClient struct {
 	storepb.StoreClient
 	store *store.TSDBStore
+	desc  string
 }
 
 func newLocalClient(c storepb.StoreClient, store *store.TSDBStore) *localClient {
+	mint, maxt := store.TimeRange()
 	return &localClient{
 		StoreClient: c,
 		store:       store,
+		desc: fmt.Sprintf(
+			"LabelSets: %v MinTime: %d MaxTime: %d",
+			labelpb.PromLabelSetsToString(labelpb.ZLabelSetsToPromLabelSets(store.LabelSet()...)), mint, maxt,
```

Comment:
I wonder if we have a way to avoid `fmt.Sprintf` here. I see it performs poorly and show up in CPU profiles

Reply:
What are the alternatives?

Reply:
Isnt it mitigated by this PR anyhow?

Reply:
Yeah, this PR together with caching matchers reduced CPU usage by 10% on every single receiver node in our case